webdriver/tests/element_click/element_click.py

```diff
@@ -0,0 +1,82 @@
+import pytest
+
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+def click(session, element):
+    return session.transport.send("POST", "session/{session_id}/element/{element_id}/click"
+                                  .format(session_id=session.session_id,
+                                          element_id=element.id))
+
+
+# 14.1 Element Click
+def test_scroll_into_view(session):
+    # 14.1 Step 4
+    session.url = inline("""<input type=text value=Federer 
+                            style="position: absolute; left: 0vh; top: 500vh">""")
+
+    element = session.find.css("input", all=False)
+    response = click(session, element)
+    assert_success(response)
+
+    # Check if element clicked is scrolled into view
+    assert session.execute_script("""
+        rect = document.getElementsByTagName("input")[0].getBoundingClientRect();
+        return rect["top"] >= 0 && rect["left"] >= 0 &&
+            (rect["top"] + rect["height"]) <= window.innerHeight &&
+            (rect["left"] + rect["width"]) <= window.innerWidth;""") is True
+
+
+@pytest.mark.parametrize("transform", ["translate(-100px, -100px)", "rotate(50deg)"])
+def test_element_not_interactable(session, transform):
```

Comment:
This test should probably have a better name as there is no way to distinguish it from the following test. Maybe `test_element_not_interactable_css_transform` or something?

Reply:
fixed